spec/csr_spec.rb

```diff
@@ -285,7 +290,7 @@
   it "loads a csr with load_from_file" do
     path = File.dirname(__FILE__) + '/fixtures/csr1.pem'
     csr = R509::CSR.load_from_file path
-    expect(csr.message_digest.name).to eq('sha1')
+    expect(csr.message_digest.name).to eq('sha256')
```

Comment:
What's the reason for changing csr1? It's just a test fixture. Were there no sha256 CSRs?